api/logs/views.py

```diff
@@ -0,0 +1,216 @@
+from modularodm import Q
+from rest_framework import generics, permissions as drf_permissions
+from rest_framework.exceptions import NotFound
+
+from website.models import Node, NodeLog
+
+from framework.auth.oauth_scopes import CoreScopes
+
+from api.base.filters import ODMFilterMixin
+from api.base.utils import get_user_auth
+from api.base import permissions as base_permissions
+from api.logs.serializers import NodeLogSerializer
+from api.nodes.serializers import NodeSerializer
+from api.nodes.utils import get_visible_nodes_for_user
+
+class LogList(generics.ListAPIView, ODMFilterMixin):
```

Comment:
If we don't need a global watch list, and if you are correct that we don't have a use case, I'm happy to remove that view.